server/routes/debug.ts

```diff
@@ -57,13 +119,17 @@ export default function makeDebugRoutes(services: Services): Record<string, Requ
         pageNumber = 1
       }
 
-      // Get events from API
-      const eventsResponse = await incidentReportingApi.getEvents({
+      const orderString = order as string
+      // Get reports from API
+      const reportsResponse = await incidentReportingApi.getReports({
         prisonId,
-        eventDateFrom: fromDate,
-        eventDateUntil: toDate,
+        incidentDateFrom: fromDate,
+        incidentDateUntil: toDate,
+        type: incidentType,
+        status: incidentStatuses,
         page: pageNumber - 1,
-        // sort: ['eventDateAndTime,ASC'],
+        size: 5,
```

Comment:
i would suggest linking to `server/data/incidentReportingApi.ts:defaultPageSize` so that we use a single number throughout

Reply:
I will remove this, accidentally left that in from when I was changing it for testing purposes